client/src/components/FilterMenu.js

```diff
@@ -1,16 +1,18 @@
 import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import PlacesAutocomplete, {
-  geocodeByAddress,
-  getLatLng,
-} from 'react-places-autocomplete';
+import { bool, func } from 'prop-types';
+import PlacesAutocomplete, { geocodeByAddress, getLatLng } from 'react-places-autocomplete';
 import '.././css/FilterMenu.css';
 
+const propTypes = {
+  visibility: bool.isRequired,
+  changeSalary: func.isRequired,
+  changeLocation: func.isRequired,
+};
+
```

Comment:
Moving propTypes to the top for visibility: https://github.com/airbnb/javascript/tree/master/react#ordering